kornia/geometry/transform/imgwarp.py

```diff
@@ -36,6 +36,7 @@ def warp_perspective(
     dsize: Tuple[int, int],
     mode: str = 'bilinear',
     padding_mode: str = 'zeros',
+    padding_tensor: torch.Tensor = torch.zeros(3),  # needed for jit
```

Comment:
```suggestion
    padding_tensor: Optional[torch.Tensor] = torch.zeros(3),  # needed for jit
```
because in below you check if it's `None`

Reply:
This will give 
```
E           RuntimeError: 
E           'Optional[Tensor]' object has no attribute or method 'shape'.:
E             File "/Users/raphaelvorias/projects/kornia/kornia/geometry/transform/imgwarp.py", line 99
E               # fill padding is only supported for 3 channels because we can't set padding_tensor default
E               # to None as this gives jit issues.
E               if padding_mode == "fill" and padding_tensor.shape != torch.Size([3]):
E                                             ~~~~~~~~~~~~~~~~~~~~ <--- HERE
E                   raise ValueError(f"Padding_tensor only supported for 3 channels. Got {padding_tensor.shape}")
```